virtcontainers/qemu_arch_base.go

```diff
@@ -257,7 +257,7 @@ func (q *qemuArchBase) bridges(number uint32) []types.PCIBridge {
 func (q *qemuArchBase) cpuTopology(vcpus, maxvcpus uint32) govmmQemu.SMP {
 	smp := govmmQemu.SMP{
 		CPUs:    vcpus,
-		Sockets: vcpus,
+		Sockets: maxvcpus,
```

Comment:
(thought I'd left this question already - maybe it was on a parallel PR or Issue???)
@devimc - does this have any performance impacts? iirc, way back when we first played with topologies there was discussion that different topologies could affect performance. Did you measure with the metrics report by chance?

Reply:
@grahamwhaley metrics-CI is happy, do you want me to run a specific test?